code/modules/mob/living/say.dm

```diff
@@ -269,13 +269,15 @@ var/list/headset_modes = list(
 			rendered_message = replacetext(rendered_message, T, "<i style='color: red;'>[T]</i>")
 
 	//AI mentions
-	if(istype(src, /mob/living/silicon/ai) && speech.frequency && speech.job != "AI")
+	if((isAI(src)  || isshell(src)) && speech.frequency && !(findtext(speech.job,"AI")<>0) && (speech.name != src.name))
```

Comment:
Wait no why is this necessary? !findtext(speech.job, "AI") should do the same thing as !(findtext(speech.job, "AI")<>0). !var returns true if var is 0 or null. Additionally, <> is identical to the != operator and shouldn't be used for the sake of consistency.

Reply:
ye, id sworn i had issues with it that way earlier. committed.